### PR TITLE
Adding Pharmacokinetic parameters

### PR DESCRIPTION
**Pull request by meslami-netrias**
Adding in pharmacokinetic parameters to the dataset and updating the README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -70,6 +70,20 @@ diseaseName<sup>3</sup> | Name of the disease
 diseaseType<sup>3</sup> | DisGeNET disease type: disease, phenotype and group
 diseaseSemanticType<sup>3</sup> | UMLS Semantic Type(s) of the disease
 Score<sup>3</sup> | DisGENET score for the Gene-Disease association
+CAS number<sup>5</sup> | Unique numerical identifier assigned by the Chemical Abstracts Service (CAS) to every chemical substance described in the open scientific literature
+year of approval<sup>5</sup> | Year drug approved
+Status<sup>5</sup> | Label if the drug is discontinued
+routes of administration<sup>5</sup> | Route of administration 
+Volume of distribution (VD)<sup>5</sup> | Distribution volume in liters
+Clearance (Cl)<sup>5</sup> | volume of blood, serum, or plasma completely cleared of drug per unit of time (liter/hour)
+Plasma Protein Binding (PPB)<sup>5</sup> | Plasma proteein binding assay
```
Comment (mwes) [inline README.md]: Fix spelling - protein

File: README.md [hunk 2]
```diff
@@ -70,6 +70,20 @@ diseaseName<sup>3</sup> | Name of the disease
...
+Half-life (t1/2)<sup>5</sup> | Half-life in 
+F<sup>5</sup> | Bioavailability as a percetnage
```
Comment (mwes) [inline README.md]: Fix spelling - percentage

File: README.md [hunk 3]
```diff
@@ -70,6 +70,20 @@ diseaseName<sup>3</sup> | Name of the disease
...
 diseaseSemanticType<sup>3</sup> | UMLS Semantic Type(s) of the disease
 Score<sup>3</sup> | DisGENET score for the Gene-Disease association
+CAS number<sup>5</sup> | Unique numerical identifier assigned by the Chemical Abstracts Service (CAS) to every chemical substance described in the open scientific literature
+year of approval<sup>5</sup> | Year drug approved
+Status<sup>5</sup> | Label if the drug is discontinued
+routes of administration<sup>5</sup> | Route of administration 
+Volume of distribution (VD)<sup>5</sup> | Distribution volume in liters
+Clearance (Cl)<sup>5</sup> | volume of blood, serum, or plasma completely cleared of drug per unit of time (liter/hour)
+Plasma Protein Binding (PPB)<sup>5</sup> | Plasma proteein binding assay
+Half-life (t1/2)<sup>5</sup> | Half-life in
```
Comment (mwes) [inline README.md]: Half life in what units?

Comment (mwes) [reply README.md]: What does (t1/2) mean?

**Review by mwes — CHANGES_REQUESTED** [review]
Fix spelling and address comments in README.md

File: README.md (hunk 3 above)
Comment (meslami-netrias) [reply README.md]: half-life.

**Review by mwes — APPROVED** [review]

**Pull request merged**